### PR TITLE
Tweak Typeahead/Dropdown styling

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Changed

- Tweak Typeahead/Dropdown styling

---

Ticket: https://jira.uitdatabank.be/browse/III-5228

Not entirely sure I got this one right, the ticket mentions Dropdowns but I think by that it meant Typeaheads which don't use the same components but have overlapping classes, so I'm not sure which of the changes were meant for which (or both), so might need some tweaks

### CODE REVIEW

File: src/pages/steps/PlaceStep.tsx [hunk 1]
```diff
@@ -162,7 +162,13 @@ const PlaceStep = ({
                                 {placeName}
                               </Highlighter>
                             </Text>
-                            <Text>{streetAddress}</Text>
+                            <Text
+                              css={`
+                                color: ${colors.grey6};
+                              `}
```
Comment (simon-debruijn) [inline src/pages/steps/PlaceStep.tsx:168]:
```suggestion
                            <Text
                              color={({theme}) => theme.colors.grey6}
```

File: src/pages/steps/PlaceStep.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ import { Inline } from '@/ui/Inline';
 import type { StackProps } from '@/ui/Stack';
 import { getStackProps, Stack } from '@/ui/Stack';
 import { Text } from '@/ui/Text';
-import { getValueFromTheme } from '@/ui/theme';
+import { colors, getValueFromTheme } from '@/ui/theme';
```
Comment (simon-debruijn) [inline src/pages/steps/PlaceStep.tsx:24]:
```suggestion
import { getValueFromTheme } from '@/ui/theme';
```

See comment below

File: src/ui/Typeahead.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import type { BoxProps } from './Box';
 import { Box, getBoxProps } from './Box';
 import { InputType } from './Input';
 import {
+  colors,
```
Comment (simon-debruijn) [inline src/ui/Typeahead.tsx:10]:
```suggestion
```

File: src/ui/Typeahead.tsx [hunk 4]
```diff
@@ -118,6 +119,10 @@ const Typeahead: TypeaheadFunc = forwardRef(
             padding: 0.375rem 0.9rem;
           }
 
+          .dropdown-item {
+            border-bottom: 1px solid ${colors.grey1};
+          }
```
Comment (simon-debruijn) [inline src/ui/Typeahead.tsx:124]:
```suggestion
          .dropdown-item {
            border-bottom: 1px solid ${({ theme }) => theme.colors.grey1};
          }
```

File: src/pages/steps/PlaceStep.tsx [hunk 5]
```diff
@@ -157,13 +157,23 @@ const PlaceStep = ({
                           mainLanguage,
                         );
                         return (
-                          <Stack>
+                          <Stack
+                            css={`
+                              .address {
+                                color: ${({ theme }) => theme.colors.grey6};
```
Comment (brampauwelyn) [inline src/pages/steps/PlaceStep.tsx:163]: Is there a reason why you use an arrow function here?
You could simply write it like: 

```suggestion
                                color: ${theme.colors.grey6};
```

File: src/ui/Typeahead.tsx [hunk 6]
```diff
@@ -118,6 +118,10 @@ const Typeahead: TypeaheadFunc = forwardRef(
             padding: 0.375rem 0.9rem;
           }
 
+          .dropdown-item {
+            border-bottom: 1px solid ${({ theme }) => theme.colors.grey1};
```
Comment (brampauwelyn) [inline src/ui/Typeahead.tsx:122]: is there a reason why you use an arrow function?
You could simply write it like
```suggestion
            border-bottom: 1px solid ${theme.colors.grey1};
```

**Review by brampauwelyn — APPROVED** [review]

File: src/pages/steps/PlaceStep.tsx (hunk 5 above)
Comment (simon-debruijn) [reply src/pages/steps/PlaceStep.tsx:163]: Yes theme is not imported but injected via the arrow function (styled-components + own component library magic)

File: src/ui/Typeahead.tsx (hunk 6 above)
Comment (simon-debruijn) [reply src/ui/Typeahead.tsx:122]: Yes theme is not imported but injected via the arrow function (styled-components + own component library magic)

**Pull request merged**